packages/yum_repositories/docker-ce.repo

```diff
@@ -0,0 +1,6 @@
+[docker-ce]
```

Comment:
Why would we need Docker repositories?

Reply:
Indeed. No need for this one AFAIK

Reply:
Oops, leftover, in the beginning I thought I'll use it for containerd but after i saw that it's in epel. I'll change it.